src/js/events.js

```diff
@@ -196,133 +196,147 @@ var Events;
-                    this.attachDOMEvent(element, 'mouseover', this.handleMouseover.bind(this));
-                }, this);
-                this.listeners[name] = true;
-                break;
-            case 'editableDrag':
-                // Detecting dragover and dragleave on the contenteditables
-                this.base.elements.forEach(function (element) {
-                    this.attachDOMEvent(element, 'dragover', this.handleDragging.bind(this));
-                    this.attachDOMEvent(element, 'dragleave', this.handleDragging.bind(this));
-                }, this);
-                this.listeners[name] = true;
-                break;
-            case 'editableDrop':
-                // Detecting drop on the contenteditables
-                this.base.elements.forEach(function (element) {
-                    this.attachDOMEvent(element, 'drop', this.handleDrop.bind(this));
-                }, this);
-                this.listeners[name] = true;
-                break;
-            case 'editablePaste':
-                // Detecting paste on the contenteditables
-                this.base.elements.forEach(function (element) {
-                    this.attachDOMEvent(element, 'paste', this.handlePaste.bind(this));
-                }, this);
-                this.listeners[name] = true;
-                break;
+                    this.listeners[name] = true;
+                    break;
+                case 'editableClick':
+                    // Detecting click in the contenteditables
+                    this.base.elements.forEach(function (element) {
+                        this.attachDOMEvent(element, 'click', this.handleClick.bind(this));
+                    }.bind(this));
+                    this.listeners[name] = true;
+                    break;
+                case 'editableBlur':
+                    // Detecting blur in the contenteditables
+                    this.base.elements.forEach(function (element) {
+                        this.attachDOMEvent(element, 'blur', this.handleBlur.bind(this));
+                    }.bind(this));
+                    this.listeners[name] = true;
+                    break;
+                case 'editableKeypress':
+                    // Detecting keypress in the contenteditables
+                    this.base.elements.forEach(function (element) {
+                        this.attachDOMEvent(element, 'keypress', this.handleKeypress.bind(this));
+                    }.bind(this));
+                    this.listeners[name] = true;
+                    break;
+                case 'editableKeyup':
+                    // Detecting keyup in the contenteditables
+                    this.base.elements.forEach(function (element) {
+                        this.attachDOMEvent(element, 'keyup', this.handleKeyup.bind(this));
+                    }.bind(this));
+                    this.listeners[name] = true;
+                    break;
+                case 'editableKeyupEnter':
```

Comment:
It would probably be best to not create `editableKeyupEnter` and `editableKeyupTab` until we actually have a use for them.  

I'm also thinking we might be able to avoid creating `editableKeyupDelete` as per my other comment about using `editableKeypress`.